src/Network/CorsBuilder.php

```diff
@@ -14,6 +14,8 @@
  */
 namespace Cake\Network;
 
+use Cake\Http\Response;
```

Comment:
without `use Cake\Http\Respons` the `__construct` method expected the old class (received old class)
and will throw  php fatal error  in  future (when switching to 4.X) because we not using the old alias on that time
this is Ok with change it to
`public function __construct(HttpResponse $response, $origin, $isSsl = false)`
but i'm not sure for BC
we can change it Or this make BC issue?